client/jetpack-cloud/sections/partner-portal/payment-methods/assignment-processor-functions.ts

```diff
@@ -13,12 +13,21 @@ interface Props {
 	translate: ReturnType< typeof useTranslate >;
 	stripe: Stripe | null;
 	stripeConfiguration: StripeConfiguration | null;
+	setupIntentId: string | undefined;
 	element: StripeCardNumberElement | undefined;
 	dispatch: CalypsoDispatch;
 }
 
 export async function assignNewCardProcessor(
-	{ useAsPrimaryPaymentMethod, translate, stripe, stripeConfiguration, dispatch, element }: Props,
```

Comment:
This function does not appear to be used anywhere, so changing its signature should be safe.